src/Tribe/Aggregator/Record/List_Table.php

```diff
@@ -97,6 +97,31 @@ public function prepare_items() {
 
 		$args['posts_per_page'] = $per_page;
 
+		if ( 'scheduled' === $this->tab->get_slug() && isset( $_REQUEST['s'] ) && ! empty( $_REQUEST['s'] ) ) {
+			// nonce check if search form submitted
+			$nonce_check = isset( $_POST['s'] ) && isset( $_POST['aggregator']['nonce'] ) ? wp_verify_nonce( $_POST['aggregator']['nonce'], 'aggregator_' . $this->tab->get_slug() . '_request' ) : 1;
```

Comment:
:warning: Detected usage of a non-sanitized input variable: $_POST['aggregator']['nonce']

<sub>hash: 487dfb7b215c2ad50e0e4c9ba4121fbc</sub>